x/pdv/types/params.go

```diff
@@ -14,21 +14,37 @@ const (
 
 var (
 	DefaultSupervisors = make([]string, 0)
+	DefaultMinGasPrice = sdk.NewDecCoinFromDec("stake", sdk.MustNewDecFromStr("0.025"))
```

Comment:
should it be udec?

Reply:
Renamed to udec, anyway it should be set in genesis.json